README.md

```diff
@@ -1 +1,13 @@
-# image-lambda
\ No newline at end of file
+# Lab: AWS: S3 and Lambda
+
+## Description
+I am using Lambda on my project which allows my code to function without provisioning or managing your own server. I am also using S3 as a bucket to upload images so that they can be turned into thumbnail sized output.
+
+## Difficulties
+I ran into many difficulties and my patience was tested to say the least using the AWS CLI to create my Lambda function and invoke it. Being aware of what you are inputting into the command line and making sure all your AWS microservices are aligned is very important.
+
+## Thumbnail Image
+![Elephants](./assets/elephantsThumbnail.jpg);
+
+## Full Image
+![Elephants](./assets/elephants.jpg);
```

Comment:
In your README could you please link the tutorial that you used for this please!